README.markdown

```diff
@@ -22,3 +22,51 @@ long the outcome is distributed under the same license. Otherwise, you must
 obtain a [commercial license](./LICENSE-COMMERCIAL) that removes such
 restrictions. Read more about our licensing approach
 [here](https://www.sourcemeta.com/licensing/).
+
+Example
+--------------
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()
+{
+    // creating json using parse
+    sourcemeta::jsontoolkit::JSON _template_ =
```

Comment:
You shouldn't put underscores before/after variables. Also this is not a template. It is a document